unfoldingWord/translationCore
# Fixed gitSync

#### This pull request addresses:
Addresses https://github.com/unfoldingWord-dev/translationCore/issues/1503
and https://github.com/unfoldingWord-dev/translationCore/issues/1537



#### How to test this pull request:
Sign into your Door43 account.
Load a project into tC
On git.door43.org make a change to your project you have loaded into tC
Make a change to the exact same verse that you changed on the git server inside of tC
Try and upload that project from the projects screen.
You should now get an error message that the git history is out of sync and the upload could not be completed.

To test https://github.com/unfoldingWord-dev/translationCore/issues/1537 : 
Login as a Door43 user
Logout and then login as local user
logout then log back in as a door43 user
Try and upload a project. You should no longer recieve the "wrong username or password" error.

**NOTE** GitSync file is no longer in use. This is all done from the recentProjectActions now. We tried to make it work but it wasnt happening.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1567)
